jetson/percep/pcl.cpp

```diff
@@ -279,27 +279,28 @@ double PCL::FindClearPath(const std::vector<std::vector<int>> &interest_points,
     std::vector<int> obstacles; //index of the leftmost and rightmost obstacles in path
     
     //Check Center Path
-    if(CheckPath(interest_points, viewer, obstacles, compareLine(0,-HALF_ROVER), compareLine(0,HALF_ROVER))) {
+    if(CheckPath(interest_points, viewer, obstacles, compareLine(0,-HALF_ROVER), compareLine(0,HALF_ROVER))){
         std::cout << "CENTER PATH IS CLEAR!!!" << std::endl;
```

Comment:
see above: are debug print statements necessary?

Reply:
As per what Matt told me, since I didn't add this particular debug print statement, I won't delete this specific one.